controllers/validators/schema.js

```diff
@@ -75,6 +75,40 @@ module.exports = {
       additionalProperties: false
     }
   },
+  newUsersWithMyTags: {
+    id: 'newUsersWithMyTags',
+    query:{
+      properties:{
+        sort: {
+          type: 'string',
+          pattern: '^-created$'
```

Comment:
there is an `enum` option which accepts a list of allowed values. Or even [`const`](http://json-schema.org/latest/json-schema-validation.html#rfc.section.6.24) which checks equality.